tuxemon/locale.py

```diff
@@ -65,6 +65,11 @@ class LocaleInfo:
     domain: str
     path: str
 
+class gettingMoney:
```

Comment:
Class named should be TitleCase and would usually be a noun.

e.g. MoneyInterface, MoneyInfo

The method should have the @staticmethod decorator applied to it.

But this class isn't even necessary. You shouldn't wrap functions in classes just for the sake of it.

The function itself is a bit borderline, as it's so simple, but it will reduce the refactoring required if the implementation of the money dict changes... So I guess it's ok.